tests/ActivityLoggerTest.php

```diff
@@ -26,6 +26,14 @@
     expect($this->getLastActivity())->toBeNull();
 });
 
+it('will log activity with a null log name', function () {
+    config(['activitylog.default_log_name' => null]);
+
+    activity()->log($this->activityDescription);
+
+    expect($this->getLastActivity()->description)->toEqual($this->activityDescription);
```

Comment:
Shouldn't you also expect the `log_name` to be `null`!? 🤔

Reply:
Apologies, merge conflicts overwrote the test. I've resolved this 👍